README.adoc

```diff
@@ -290,7 +322,7 @@ the `spring-cloud-starter-sleuth` module to your project.
               <dependency>
                   <groupId>org.springframework.cloud</groupId>
                   <artifactId>spring-cloud-dependencies</artifactId>
-                  <version>Brixton.RELEASE</version>
+                  <version>Dalston.BUILD-SNAPSHOT</version>
```

Comment:
PS is the dalston change here bc you need a dependency, or just to make sure the change doesn't affect brixton?